UOP1_Project/Assets/Polybrush Data/Color Palette.meta

```diff
@@ -1,8 +0,0 @@
-fileFormatVersion: 2
```

Comment:
Why this file was deleted?

Reply:
I have no idea 🤔, I made sure to just commit the two .cs files I worked on

Reply:
Could you revert this file?

Reply:
> Could you revert this file?

I've re-added the missing files by cherrypicking. Now the only files listed on the PR are the ones I actually added to the project